pkg/migrations/sql/preinstall/013-telemetry.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS SCHEMA_PS_CATALOG.promscale_instance_information (
+    uuid                                                UUID NOT NULL PRIMARY KEY,
+    last_updated                                        TIMESTAMPTZ NOT NULL,
+    promscale_ingested_samples_total                    BIGINT DEFAULT 0,
+    promscale_metrics_queries_executed_total            BIGINT DEFAULT 0,
+    promscale_metrics_queries_timedout_total            BIGINT DEFAULT 0,
+    promscale_metrics_queries_failed_total              BIGINT DEFAULT 0,
+    promscale_trace_query_requests_executed_total       BIGINT DEFAULT 0,
+    promscale_trace_dependency_requests_executed_total  BIGINT DEFAULT 0,
+    deletable                                           BOOLEAN DEFAULT TRUE
```

Comment:
weren't we going to rename this column to `is_reset_row` or something like that?